tools/nni_cmd/config_schema.py

```diff
@@ -34,10 +34,7 @@
 'useAnnotation': bool,
 'tuner': Or({
     'builtinTunerName': Or('TPE', 'Random', 'Anneal', 'Evolution', 'SMAC', 'BatchTuner', 'GridSearch'),
-    'classArgs': {
-        'optimize_mode': Or('maximize', 'minimize'),
-        Optional('speed'): int
-        },
+    Optional('classArgs'): dict,
```

Comment:
since the spec of tpe/smac is different from random/gridsearch/batchtuner, is it possible to check their spec separately?